rollup/README.md

```diff
@@ -35,3 +35,16 @@ make rollup_bins
 ./build/bin/gas_oracle --config ./conf/config.json
 ./build/bin/rollup_relayer --config ./conf/config.json
 ```
+
+### Running unit tests
+
+Follow these steps to run unit tests, in the repo's root dir:
+
+```
+wget https://github.com/scroll-tech/da-codec/releases/download/v0.0.0-rc0-ubuntu20.04/libzktrie.so
+sudo mv libzktrie.so /usr/local/lib
```

Comment:
put libzktrie.so to /usr/local/lib is a bit peculiar. and it's unnecessary to `export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib` since `/usr/local/lib` is the default lib search dir.

Reply:
interesting. if I don't do `export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib`, will show lib not found. will double-check it.

Reply:
need to set `LD_LIBRARY_PATH`.